frontend/src/metabase/query_builder/components/view/View/QueryViewNotebook/QueryViewNotebook.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from "prop-types";
 import Notebook from "metabase/query_builder/components/notebook/Notebook";
 import { NotebookContainer } from "./QueryViewNotebook.styled";
 
-const delayBeforeNotRenderingNotebook = 300;
+const delayBeforeNotRenderingNotebook = 10;
```

Comment:
This can't be 0 because some events got fired at the end of the transition.

Reply:
The var name is a bit confusing to me. Isn't it just an animation duration? 🤔
Also, should we maybe use uppercase + snake case as it's a constant?

Reply:
The duration of the animation is exactly the time it takes ("delay") until the notebook shows up.
I agree that this is very confusing. Likely I will have to refactor the surrounding code anyway in master but for this release branch, I try to make the change as small as possible.